app/controllers/debates/votes_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Debates
+  class VotesController < ApplicationController
```

Comment:
I think we're missing a call to the `authenticate_user!` method here :thinking:. A couple of tests for this case:

```ruby
describe Debates::VotesController do
  describe "POST create" do
    it "does not authorize unauthenticated users" do
      debate = create(:debate)

      post :create, xhr: true, params: { debate_id: debate.id, value: "yes" }

      expect(response).to be_unauthorized
    end

    it "redirects unauthenticated users without JavaScript to the sign in page" do
      debate = create(:debate)

      post :create, params: { debate_id: debate.id, value: "yes" }

      expect(response).to redirect_to new_user_session_path
    end
```

Same in the legislation proposals votes controller.